CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Changelog
 
+## [1.58](https://github.com/brave/brave-ios/releases/tag/v1.58)
+
+### Web3
+
+- Added Coinbase Pay on-ramp for Brave Wallet buy flows. [(#7880](https://github.com/brave/brave-ios/issues/7880))
+- Added Stripe on-ramp for Brave Wallet buy flows. ([#7794](https://github.com/brave/brave-ios/issues/7794))
+- Added Filecoin support in Brave Wallet. ([#7809](https://github.com/brave/brave-ios/issues/7809))
+- Added the ability for users to choose which Solana account to choose when connecting to Solana DApps. ([#7637](https://github.com/brave/brave-ios/issues/7637))
+- Updated default buy currency and removed field placeholder for Stripe on ramp flow. ([#7709](https://github.com/brave/brave-ios/issues/7709))
```

Comment:
Maybe reword to `Removed placeholder value in "Buy" screen and updated default currency to USD`

Placeholder is removed for all on-ramp not just for Stripe so no need to mention the provider here.